README.md

```diff
@@ -1 +1,41 @@
-# Emby.Theater.Electron
+# Jellyfin Theater
+
+Jellyfin Theater is a local client that connects to a Jellyfin server. 
+Currently this takes place by contacting https://
+# Building
+
+This application is implemented as an electron app and is built off of 
+a NodeJS code base. Because of this you will need to NodeJS runtime and
+package manager. The following versions have been tested: 
+
+| Software Name | Version          |
+| ------------- | ---------------- |
+| Node JS       | 11.5.0           |
+| npm           | 6.4.1            |
+| ------------- | ---------------- |
+
+
+Take the following steps to build the application
+
+```
+# Get the source
+git clone git@github.com:jellyfin/jellyfin-theater-electron.git
```

Comment:
Might be better to give https repo link instead of of git - it's way easier to get to that through proxies.

Reply:
Just sent a pr for this issue.